lib/actions/device.coffee

```diff
@@ -208,16 +207,14 @@ exports.await =
 		options.interval ?= 3000
 
 		poll = ->
-			resin.models.device.isOnline params.name, (error, isOnline) ->
-				return done(error) if error?
-
+			resin.models.device.isOnline(params.uuid).then (isOnline) ->
```

Comment:
This can be:
```coffee
		poll = ->
			resin.models.device.isOnline(params.uuid).then (isOnline) ->
				if isOnline
					console.info("Device became online: #{params.uuid}")
					return
				else
					console.info("Polling device network status: #{params.uuid}")
					return Promise.delay(options.interval).then(poll)
		poll().nodeify(done)
```